Assets/Scripts/Login/Networking/LoginServerClient.cs

```diff
@@ -0,0 +1,271 @@
+        {
+            ValidateToken("Save Character");
+            monoBehaviour.StartCoroutine(HandlePayloadRequest(
+                UnityWebRequest.kHttpVerbPOST,
+                connectionAddress + Post_CreateCharacter,
+                JsonUtility.ToJson(characterRequest),
+                "Save Character", 
+                true,
+                resultCallback
+            ));
+        }
+
+        /// <summary>
+        /// CentCom call to delete a character of the current user.
+        /// Requires Authenticate to have been called at least once before.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="resultCallback"></param>
+        public void DeleteCharacter(string id, Action<string, bool> resultCallback)
+        {
+            ValidateToken("Delete Character");
+            monoBehaviour.StartCoroutine(HandleSimpleRequest(
+                UnityWebRequest.kHttpVerbDELETE,
+                connectionAddress + Delete_DeleteCharacter + $"/{id}",
+                "Delete Character", 
+                true,
+                resultCallback
+            ));
+        }
+
+        /// <summary>
+        /// Handles requests that do not contain a data payload
+        /// </summary>
+        /// <param name="method">A value from UnityWebRequest.kHttpVerb<...></param>
+        /// <param name="url">The endpoint url where the request will be sent</param>
+        /// <param name="name">Name of this request. For logging purposes only.</param>
+        /// <param name="authenticatedRequest">Set to true, if the request requires authentication</param>
+        /// <param name="resultCallback">Callback to be executed when a response is received</param>
+        private IEnumerator HandleSimpleRequest(string method, string url, string name, bool authenticatedRequest, Action<string, bool> resultCallback)
+        {
+            Func<string, UnityWebRequest> requestCall;
+
+            switch (method)
+            {
+                case UnityWebRequest.kHttpVerbGET:
+                    requestCall = UnityWebRequest.Get;
+                    break;
+                case UnityWebRequest.kHttpVerbDELETE:
+                    requestCall = UnityWebRequest.Delete;
+                    break;
+                default:
+                    Debug.LogError($"Unknown request method {method}");
+                    yield break;
+            }
+            
+            using (UnityWebRequest webRequest = requestCall(url))
+            {
+                webRequest.SetRequestHeader("Content-Type", "application/json");
+                if (authenticatedRequest)
+                {
+                    webRequest.SetRequestHeader("Authorization", $"Bearer {token}");
+                }
+                yield return webRequest.SendWebRequest();
+                if (webRequest.isNetworkError || webRequest.isHttpError)
+                {
+                    Debug.Log($"{name} request error: {webRequest.error}");
```

Comment:
This should be `Debug.LogError` or at least  `Debug.LogWarning`

Reply:
I'd go with LogWarning, as LogError indicates to me that the code couldn't recover from the error

Reply:
Well, it couldn't presumably. You got an error and no action was taken to recover at this point in the code

Reply:
No action will be taken because the request was not successful, either due to server error or a malformed request. But it does not freeze or crash the game. I'll change to a warning.

Reply:
A warning implies that the game will continue running roughly as expected
If the request was not successful for example when you get the character list, how do you continue without a list of characters?

Reply:
If the character screen does not receive a character list, which it wont because of the error, it defaults to saying no characters were found.

While the output may not be entirely accurate, it does not break the UI or stop the player. The player may attempt to re-log and load the characters again, or create a new character, which also may work and would again reload the character list.

All the screens behave in a similar way.